src/components/tasks/card/index.tsx

```diff
@@ -93,8 +93,17 @@ const Card: FC<CardProps> = ({
 
     const [showSuggestion, setShowSuggestion] = useState<boolean>(false);
 
-    const localStartedOn = new Date(parseInt(cardDetails.startedOn, 10) * 1000);
-    const fromNowStartedOn = moment(localStartedOn).fromNow();
+    function getStartedAgo() {
+        if (cardDetails.startedOn === undefined) {
+            return 'N/A';
+        } else {
+            const localStartedOn = new Date(
+                parseInt(cardDetails.startedOn, 10) * 1000
+            );
+            const fromNowStartedOn = moment(localStartedOn).fromNow();
```

Comment:
i do see issues here @Maheima

Reply:
where?

Reply:
localStartedOn, in which time zone format we are saving date, to TUC, IST or ....